notary/notary.spec

```diff
@@ -0,0 +1,310 @@
+ln -s ../../.. src/%{import_path}
+%if ! 0%{?with_bundled}
+export GOPATH=$(pwd):%{gopath}
+%else
+export GOPATH=$(pwd):$(pwd)/Godeps/_workspace:%{gopath}
+%endif
+
+%{!?gobuild:%global gobuild go build -compiler gc -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n')" -a -v -x}
+
+export LDFLAGS="-w -X github.com/docker/notary/version.GitCommit %{commit} -X github.com/docker/notary/version.NotaryVersion $(cat NOTARY_VERSION)"
+%gobuild -tags pkcs11 -o bin/notary-server ./cmd/notary-server
+%gobuild -tags pkcs11 -o bin/notary-signer ./cmd/notary-signer
+%gobuild -tags pkcs11 -o bin/notary ./cmd/notary
+
+
+%install
+install -d %{buildroot}%{_bindir} %{buildroot}%{_sysconfdir}/notary
+install -p -m 755 bin/notary{,-server,-signer} %{buildroot}%{_bindir}
+install -p -m 600 fixtures/server-config.json %{buildroot}%{_sysconfdir}/notary
+install -p -m 600 fixtures/signer-config.json %{buildroot}%{_sysconfdir}/notary
+
+install -d %{buildroot}%{_unitdir}
+install -p -m 644 %{SOURCE1} %{SOURCE2} %{buildroot}%{_unitdir}
+
+chmod a-x docs/opensslCertGen.sh migrations/migrate.sh
+
+# source codes for building projects
+%if 0%{?with_devel}
+install -d -p %{buildroot}/%{gopath}/src/%{import_path}/
+echo "%%dir %%{gopath}/src/%%{import_path}/." >> devel.file-list
+# find all *.go but no *_test.go files and generate devel.file-list
+for file in $(find . -iname "*.go" \! -iname "*_test.go" | grep -v "./Godeps") ; do
+    echo "%%dir %%{gopath}/src/%%{import_path}/$(dirname $file)" >> devel.file-list
+    install -d -p %{buildroot}/%{gopath}/src/%{import_path}/$(dirname $file)
+    cp -pav $file %{buildroot}/%{gopath}/src/%{import_path}/$file
+    echo "%%{gopath}/src/%%{import_path}/$file" >> devel.file-list
+done
+%endif
+
+# testing files for this project
+%if 0%{?with_unit_test} && 0%{?with_devel}
+install -d -p %{buildroot}/%{gopath}/src/%{import_path}/
+# find all *_test.go files and generate unit-test-devel.file-list
+for file in $(find . -iname "*_test.go" | grep -v "./Godeps"); do
+    echo "%%dir %%{gopath}/src/%%{import_path}/$(dirname $file)" >> devel.file-list
+    install -d -p %{buildroot}/%{gopath}/src/%{import_path}/$(dirname $file)
+    cp -pav $file %{buildroot}/%{gopath}/src/%{import_path}/$file
+    echo "%%{gopath}/src/%%{import_path}/$file" >> unit-test-devel.file-list
+done
+%endif
+
+%if 0%{?with_devel}
+sort -u -o devel.file-list devel.file-list
+%endif
+
+%check
+%if 0%{?with_check} && 0%{?with_unit_test}
+%if ! 0%{?with_bundled}
+export GOPATH=$(pwd):%{gopath}
+%else
+export GOPATH=$(pwd):$(pwd)/Godeps/_workspace:%{gopath}
+%endif
+
+%gotest -tags pkcs11 ./...
+%endif
+
+#define license tag if not already defined
+%{!?_licensedir:%global license %doc}
+
+%if 0%{?with_devel}
+%files devel -f devel.file-list
+%license LICENSE
+%dir %{gopath}/src/%{provider}.%{provider_tld}/%{project}
+# This is not caught by the generated devel.file-list
+%dir %{gopath}/src/%{import_path}/cmd
```

Comment:
cmd directory does not contain any file with 'package cmd' directive. All files has 'package main'. For that, none of the files can be imported. This, cmd does not define importable golang package.

Please remove.